pages/blog/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <b-container data-qa="blog">
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <h1>{{ $t('blog.blogs') }}</h1>
+      </b-col>
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          data-qa="blog posts"
+        >
+          <ContentCard
+            v-for="(post, index) in posts"
+            :key="index"
+            :title="post.fields.headline"
```

Comment:
With the updated content model in place, could you update this to post.fields.name?